src/main/java/com/cinema/Main.java

```diff
@@ -57,19 +59,19 @@ public static void main(String[] args) throws AuthenticationException, Interrupt
                 (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(movieSession1);
         movieSessionService.add(movieSession2);
-        System.out.printf("All available sessions: \n");
+        log.info("All available sessions: \n");
         movieSessionService.findAvailableSessions(movie.getId(),
                 LocalDate.parse(DATE, DateTimeFormatter.BASIC_ISO_DATE))
-                .forEach(System.out::println);
+                .forEach(log::info);
 
         AutheticationService autheticationService =
                 (AutheticationService) injector.getInstance(AutheticationService.class);
         User user1 = new User();
         user1.setEmail("user@gmail.com");
         user1.setPassword("password");
-        System.out.println("User1 has been registered: "
+        log.info("User1 has been registered: "
                 + autheticationService.register(user1.getEmail(), user1.getPassword()));
-        System.out.println("User1 has logged in: "
+        log.info("User1 has logged in: "
                 + autheticationService.login(user1.getEmail(), user1.getPassword()));
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block